io_scene_godot/converters/simple_nodes.py

```diff
@@ -165,3 +168,68 @@ def export_light_node(escn_file, export_settings, node, parent_gd_node):
                           light_node, node.data, 'light')
 
     return light_node
+
+
+def export_curve_node(escn_file, export_settings, node, parent_gd_node):
+    """Export a curve to a Path node, with a child mesh."""
+    splines = node.data.splines
+    if len(splines) == 0:
+        logging.warning("Curve %s has no splines", node.name)
+        return None
+
+    spline = splines[0]
```

Comment:
Is it possible that we can export all the spline in the `node.data.splines` and only assign the active one to the Curve node? How do you think? (Btw you can get the active spline by calling `node.data.splines.active`)